portal/src/main/scala/cromwell/pipeline/datastorage/dto/user/PasswordUpdateRequest.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.pipeline.datastorage.dto.user
+
+import play.api.libs.json.{ Json, OFormat }
+
+final case class PasswordUpdateRequest(currentPassword: String, newPassword: String, repeatPassword: String)
+
+object PasswordUpdateRequest {
+  implicit val UpdatePasswordRequestFormat: OFormat[PasswordUpdateRequest] = Json.format[PasswordUpdateRequest]
```

Comment:
Let's make it camelCase (starts with a lowercase letter)

Reply:
Done